control-plane/pkg/reconciler/consumergroup/controller.go

```diff
@@ -120,7 +120,7 @@ func NewController(ctx context.Context, watcher configmap.Watcher) *controller.I
 	clientPool := clientpool.Get(ctx)
 
 	r := &Reconciler{
-		SchedulerFunc:                      func(s string) Scheduler { return schedulers[strings.ToLower(s)] },
+		SchedulerFunc:                      func(s string) (Scheduler, bool) { sched, ok := schedulers[strings.ToLower(s)]; return sched, ok },
```

Comment:
nit: would this work?
```suggestion
		SchedulerFunc:                      func(s string) (Scheduler, bool) { return schedulers[strings.ToLower(s)] },
```

Reply:
I tried that and sadly no, it looks like go complains:
```
not enough return values have (Scheduler) want (Scheduler, bool)
```